### DISCUSSION TITLE
Why doesn't cds-theme="dark" affect @clr/angular or @/clr/ui components?

### DISCUSSION THREAD

**Discussion by eduardoroliveira** [vmware/clarity#6182] [Q&A] [answered]
## Describe the bug

The theme dark has no effect in Angular project.

## How to reproduce

1. Create a standard Angular with Clarity as per documentation: `{ ng new my-awesome-app; cd my-awesome-app; ng add @clr/angular; }`
2. Import the theme to styles.scss file: `@import '~@cds/core/styles/theme.dark.min.css';`
3. Add the theme to the body in index.html file: 

```html
<!DOCTYPE html><html lang="en"><head>
  <meta charset="utf-8">
  <title>TanzuMigratorUi</title>
  <base href="/">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="icon" type="image/x-icon" href="favicon.ico">
  <link href="/node_modules/@cds/core/styles/theme.dark.min.css" rel="stylesheet">
</head>
<body cds-theme="dark">
  <app-root></app-root>
</body></html>
```

## Expected behavior

The dark theme should be applied, but instead, it does not make any effect.

## Versions

**Clarity project:**

- [ ] Clarity Core
- [ X ] Clarity Angular/UI

**Clarity version:**

- [ ] v3.x
- [ ] v4.x
- [ X ] v5.x
-  "@clr/angular": "12.0.0",
    "@clr/ui": "12.0.0",
    "@cds/core": "^5.0.0"

**Framework:**

- [ X ] Angular
- [ ] React
- [ ] Vue
- [ ] Other:

**Framework version:**
_ie: Angular 12.1.2

**Device:**

- Type: MacBook
- OS:  MacOS]
- Browser chrome
- Version  91.0.4472.164

**Comment by hippee-lee** [comment, marked as answer]
Hi @eduardoroliveira. 

`@cds/core` is a distinct package from `@clr/ui` and `@clr/angular`. They do not share any custom properties. We are investigating how we can align the CSS custom properties for colors across all of the packages: `@clr/angular, @clr/ui @cds/core`. There are two considerations that need to be taken into account:

1. timing - this will be a visually breaking change for all `@clr/ui & @clr/angular` consumers.
2. scoping the work to only address colors in a manageable way - it would affect every component since they all use color in multiple ways. Even though it is only color, doing it all at once would be a massive undertaking for one person...

We know this would benefit consumers as it would make the core themes standard across `@cds/core & @clr/angular & @clr/ui` projects so we are working through the what changes and how to make them appropriately. 

To provide value and context for the community I am going to convert this into a discussion and modify your title to be **Why doesn't cds-theme="dark" affect `@clr/angular or @/clr/ui` components?**
